### ISSUE TITLE
Integration with Email

### ISSUE THREAD

**Issue by pothedarsuhas** [bitnami-labs/kubewatch#215]
How to send notifications via email from kubewatch?

**Comment by mkmik** [comment]
The simplest way currently is to use the generic webhook and send a webhook to a tool like [Zapier](https://zapier.com/).

e.g:

```
handler:
  webhook:
    url: https://hooks.zapier.com/hooks/catch/206054/a8q8f6j/
```

Currently the webhook JSON payload only has a `"text"` field which contains a message like for example:

```
A `job` in namespace `` has been `updated`:
`default/opack-deploy-stqzt`
```

This means you can't put interesting things like the namespace or even type in the email subject.
I'll see if we can extend this webhook payload type with more interesting information.

Let me know if that's enough for your use case.

**Comment by mkmik** [comment]
Once we cut a release containing #187, you'll have access to more fields, including the object kind, name, namespace, and event reason.

**Comment by mkmik** [comment]
if external mailers like zapier etc are not suitable, perhaps a simple smtp handler will work.
do you need support for authentication, tls, etc?

**Comment by pothedarsuhas** [comment]
Yes, external mailers are not suitable in our use case, please support building smtp handler and authentication

**Comment by mkmik** [comment]
Sounds reasonable. Will update when implemented

**Comment by mkmik** [comment]
Released in kubewatch v0.1.0. Helm chart version 1.1.1 includes support for the smtp email handler config:

```
$ cat > values.yaml <<EOF
resourcesToWatch:
  daemonset: true
  job: true
slack:
  enabled: false
smtp:
  enabled: true
  to: yourteam@email.com
  from: yourmonitoringsystem@email.com
  smarthost: smtp.yourcompany.com:25
  auth:
    username: someusername
    password: somepassword
  requireTLS: true
EOF
$ helm install kw bitnami/kubewatch  -f values.yaml
```
